lib/hoe/package.rb

```diff
@@ -101,7 +101,7 @@ def install_gem name, version = nil, rdoc = true
     cmd += " #{null_dev}" unless Rake.application.options.trace
 
     puts cmd if Rake.application.options.trace
-    system cmd
+    system(cmd).tap { Gem::Specification.reset }
```

Comment:
This breaks compatibility with 1.8... and I don't like `tap` in the first place. I'll switch it around.